pkg/kubelet/images/image_manager_test.go

```diff
@@ -239,7 +337,8 @@ func pullerTestEnv(t *testing.T, c pullerTestCase, serialized bool, maxParallelI
 	backOff.Clock = fakeClock
 
 	fakeRuntime = &ctest.FakeRuntime{T: t}
-	fakeRecorder := &record.FakeRecorder{}
+	fakeRecorder = testutil.NewFakeRecorder()
+	fakeRecorder.T = t
```

Comment:
Would it be better to define a special recorder for this test instead of setting fields manually? Another alternative would be to modify NewFakeRecorder to take `t *testing.T` as a parameter.

Reply:
For now, I modified the constructor. Defining a special recorder for this test might be more proper, but I also dont see any specific problem with using this existing one.